caffe2/operators/hip/pool_op_miopen.cc

```diff
@@ -27,7 +27,7 @@ class MIOPENPoolOp : public ConvPoolOpBase<HIPContext> {
         alpha_(OperatorBase::GetSingleArgument<float>("alpha", 1.0)),
         beta_(OperatorBase::GetSingleArgument<float>("beta", 0.0)),
         do_backward_(
-            OperatorBase::GetSingleArgument<bool>("do_backward", true)),
+            OperatorBase::GetSingleArgument<bool>("do_backward", false)),
```

Comment:
I don't see this argument getting set anywhere, we can probably even remove it?

Reply:
Yes. This is unused. Removed it